clients/adapter/src/main/java/org/eclipse/hono/adapter/client/command/CommandRouterCommandConsumerFactory.java

```diff
@@ -172,7 +167,7 @@ public void registerInternalCommandConsumer(
 
         // register the command handler
         final CommandHandlerWrapper commandHandlerWrapper = new CommandHandlerWrapper(tenantId, deviceId, gatewayId,
-                commandHandler);
+                commandHandler, Vertx.currentContext());
```

Comment:
this makes it hard to test, doesn't it? Wouldn't it be better to pass in a vert.x Context instead?

Reply:
That would mean a change in the `CommandConsumerFactory` methods and would involve quite many places.
And testing with using `Vertx.currentContext()` would still be possible using the `VertxExtension` for the test.

Reply:
I see.